stitch/options.py

```diff
@@ -44,3 +47,29 @@ def validate(self, obj, value):
         if isinstance(value, str) or value is None:
             return value
         self.error(obj, value)
+
+
+class LangMapper:
```

Comment:
Could you add a short docstring here? Where are the language names coming from, the user? Are the kernel names coming from the same source as `jupyter kernelspec`.

Reply:
I guess I can. All these names come from user.

I'm not sure about their origin... That are the names that work in Stitch :-) For example in `%APPDATA%\jupyter\kernels\ir` folder I have `kernel.json` file:
```json
{
  "argv": ["C:/R/bin/x64/R", "--slave", "-e", "IRkernel::main()", "--args", "{connection_file}"],
  "display_name": "R",
  "language": "R"
}
```
`ir` works in Stitch but `R` does not.